config/index.js

```diff
@@ -312,6 +312,23 @@ var conf = convict({
     env: 'LOCKOUT_ENABLED',
     default: false
   },
+  oauth: {
+    host: {
+      doc: 'OAuth host for verifying tokens',
+      default: 'localhost',
+      env: 'OAUTH_HOST'
+    },
+    port: {
+      doc: 'OAuth port for veryfing tokens',
+      default: 9010,
+      env: 'OAUTH_PORT'
+    },
+    insecure: {
+      doc: 'Connect to OAuth server via insecure plain http',
+      default: false,
+      env: 'OAUTH_INSECURE'
+    }
+  },
```

Comment:
Are you asked whether this should just be a URL rather than individual host/port/protocol components?  If so then yes, I think that would make sense, but we'll have to figure out how that plays with the hapi oauth integration lib.